lib/create-response-handler.ts

```diff
@@ -1,46 +1,51 @@
 /**
+ * The contents of this file are only relevant if your service uses the
+ * legacy setup. A legacy setup involves connecting to GOV.UK Verify with a
+ * Matching Service Adapter (MSA).
+ *
  *
  * create-response-handler
  *
  * Helper function to make handling callbacks from
  * `passport.authenticate('verify', callback)` easier.
  *
  * This is necessary because there are a number of different scenarios
- * that relying parties need to handle when dealing with Verify responses.
- * passport.js isn't aware of these, so the interface its provides is not ideal.
+ * that services need to handle when dealing with Verify responses.
+ * passport.js isn't aware of these, so the interface it provides is not ideal.
  */
 /** */
 
-import { TranslatedMatchingResponseBody, TranslatedIdentityResponseBody, Scenario } from './verify-service-provider-api/translated-response-body'
+import { TranslatedMatchingResponseBody, Scenario } from './verify-service-provider-api/translated-response-body'
 
 /**
- * ResponseScenarios
- *
- * An object enumerating scenarios that can occur when handling a response from
- * Verify.
+ * The callbacks in the `ResponseScenarios` interface define scenarios
+ * that can occur when handling a response from GOV.UK Verify.
  *
- * Relying Parties should handle each of these scenarios appropriately.
+ * Services should handle each of these scenarios appropriately.
  */
 export interface ResponseScenarios {
+
   /**
    * Called when handling a success response from a user that was matched by your
    * matching service. Your callback should redirect the user to the appropriate
    * page so they can begin using your service.
    *
-   * The callback will be provided with a `user` object, which will be whatever
-   * was returned from your `verifyUser` callback.
+   * The callback will be provided with a `user` object, which will be the output
+   * from your `verifyUser` callback.
    */
   onMatch: (user: any) => any,
+
   /**
    * Called when handling a success response from a user that was not matched by
    * your matching service, but who has had a new account created. Your callback
    * should redirect the user to the appropriate page so they can begin using your
    * service (you may or may not want to treat new users differently to existing users).
    *
-   * The callback will be provided with a `user` object, which will be whatever
-   * was returned from your `createUser` callback.
+   * The callback will be provided with a `user` object, which will be the output
+   * from your `createUser` callback.
    */
   onCreateUser: (user: any) => any,
+
   /**
    * Called when the user failed to authenticate with Verify,
```

Comment:
```suggestion
   * Called when the user failed to authenticate with the identity provider,
```

Reply:
[as above]

Reply:
```suggestion
   * Called when the user failed to authenticate,
```
As above:
I agree on muddying the water with too much detail, so then let's leave the details out rather than putting in incorrect information. It's incorrect to say users authenticate with Verify and we shouldn't encourage the phrasing.